recipes/pyserver/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pyserver" %}
+{% set version = "1.0.2" %}
+
+package:
+   name: "{{ name }}"
+   version: "{{ version }}"
+
+source:
+   url: http://software.igwn.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+   sha256: c51c626e8d99714e5be65b3f5e081dee84489df6749cfca76f419f3861e60539
+
+build:
+  number: 0
+  noarch: python
```

Comment:
I have changed the `skip` to `noarch: python` as suggested here: https://github.com/conda-forge/staged-recipes/pull/15380#discussion_r657315781